salt/metalk8s/addons/prometheus-operator/deployed/ingress.sls

```diff
@@ -0,0 +1,25 @@
+#! metalk8s_kubernetes
+
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: prometheus-operator-alertmanager
+  namespace: metalk8s-monitoring
```

Comment:
Now that I'm thinking about it, do we **really** need the Ingress to talk to Alertmanager from Salt Master?

It's the first time that we need access to a Service from an execution module, so we didn't really have precedence, but we do look for Service Endpoints in the external pillar `metalk8s_endpoints`, and use those values in some formulas.
IMO, we should do the same here, and maybe refactor slightly to have:
- an execution module for gathering Endpoints for a Service
- this module used in the pillar module
- and use it instead of the Ingress in your `metalk8s_monitoring` module

IIRC, `requests` expose a way to pass multiple hosts to try from (but IIRC, an Endpoint is created if the Pod is Ready, so you shouldn't need it).

Reply:
Removed, I now use alertmanager service endpoint.